product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -71,7 +71,10 @@ Prior to creating your cluster, make sure you have enough resources. Without eno
     1. Select the instance series and size. See [Sizes for virtual machines in Azure](https://docs.microsoft.com/en-us/azure/virtual-machines/sizes) or [Amazon EC2 Instance Types](https://aws.amazon.com/ec2/instance-types/) for information to help you choose the appropriate instance type. 
   
 1.  In the **Storage** section, select your volume type from the **Volume Type** list. 
-    - For Azure, BigAnimal currently supports Azure Premium SSD storage types. In **Volume Properties**, select the type and amount of storage needed for your cluster. See [the Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssds) for more information. 
+    - For Azure, in **Volume Type** select either Premium SSD or Ultra Disk. Compared to Premium SSD volumes, Ultra Disks offer lower latency, high-performance options and direct control over your disk's IOPS (input/output operations per second). BigAnimal recommends using Ultra Disks for workloads that require the most demanding performance. See [Using Azure ultra disks](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-enable-ultra-ssd?tabs=azure-portal) for more information. 
+      - For Premium SSD, in **Volume Properties**, select the type and amount of storage needed for your cluster. See [Azure Premium SSD storage types](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssds) for more information. 
+
+      - For Ultra Disk, in **Volume Properties**, select the disk size and IOPS for your cluster. BigAnimal calculates the throughput based on your IOPS settings. 
```

Comment:
```suggestion
      - For Ultra Disk, in **Volume Properties**, select the disk size and IOPS for your cluster. BigAnimal calculates the throughput based on your IOPS settings. _Be careful_: while setting the required IOPS for the disk you should also consider the VM limits that are tied to the VM size that you have selected before. See [Ultra disk IOPS](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#ultra-disk-iops) for more information.
```

Reply:
Good catch, we should also show a warning in the UI (until  IOPS with VMs cap ourselves) ?

Reply:
Addressed in https://github.com/EnterpriseDB/docs/pull/2762/commits/f19ec677f23722658eec67537650b636a175788b

Reply:
> Good catch, we should also show a warning in the UI (until we validate IOPS with VMs cap ourselves) ?

I agree, this can be a useful warning I think that will improve the UX, and maybe later don't allow the user to go over the maximum value as you said